### ISSUE TITLE
submit_btn in multimodal not working as intended.

### ISSUE THREAD

**Issue by labaxisAI2** [gradio-app/gradio#8406]
I notice I was not able to hide the submit button in my multimodal textbox, but looks like in the demo it does not work as well.
Someone please let me know how to hide this?

![image](https://github.com/gradio-app/gradio/assets/160642900/71f09f1e-c958-4990-8126-ea1ad7254fa2)

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr

```

### System Info

```shell
this shows up on the gradio website as well.
```

### Severity

I can work around it

**Comment by labaxisAI2** [comment]
Anyone find a solution or work around?

**Comment by freddyaboulton** [comment]
Yes this is a bug. Would you like to open a PR for this @labaxisAI2 ? Should be a one line change [here](https://github.com/gradio-app/gradio/blob/96d8de231270321da5f310768643363276df3204/js/multimodaltextbox/shared/MultimodalTextbox.svelte#L224)

```svelte
<button class:disabled={disabled || submit_btn == false} class="submit-button" 
    on:click={handle_submit}
>{submit_btn}</button>
```

**Comment by abidlabs** [comment]
This issue is now fixed in the `5.0-dev` branch, you can test with:

```py
import gradio as gr
import time

with gr.Blocks() as demo:
    gr.MultimodalTextbox(value={"text": "hi"}, interactive=True, submit_btn=False)

demo.launch()
```
